www/_posts/2017-03-31-android-release.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova Android 6.2.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 6.2.0` has been released! 
+
+This release has fixed issues introduced by the **Android SDK Tools 25.3.1** release. **Google** dropped support for the `android` binary, so `cordova-android` has now adopted support for the `avdmanager` and `sdkmanager` binaries. We have also taken the opportunity to rewrite how we use gradle on the users system.
```

Comment:
We should explicitly state here that we now require either Gradle or Android Studio to be installed, since our old configuration didn't require either of these things to be installed previously.